entrypoint.sh

```diff
@@ -40,3 +40,33 @@ echo "Uploading wheels to anaconda.org..."
 
 anaconda -t $ANACONDA_TOKEN upload --force -u "$ANACONDA_ORG" "$INPUT_ARTIFACTS_PATH"/*.whl
 echo "Index: https://pypi.anaconda.org/$ANACONDA_ORG/simple"
+
+# Get the package name from the "${INPUT_ARTIFACTS_PATH}"/*.whl files
+if [ -d "${INPUT_ARTIFACTS_PATH}" ]; then
```

Comment:
Opting for package name discovery over making it a required argument given https://github.com/scientific-python/upload-nightly-action/issues/8#issuecomment-1565022024 stressed the idea of simplicity of use for the end user.